src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdown.ts

```diff
@@ -40,6 +43,10 @@ export class ResponsiveDropdown {
     this.bindNukeEvents();
   }
 
+  private set scrollLocked(lock: boolean) {
+    this.coveoRoot.toggleClass(ResponsiveDropdown.LOCK_SCROLL_CSS_CLASS_NAME, lock);
```

Comment:
I don't think blocking the scroll this way on the root will work in situations where the root isn't the body. Better toggle the class on the body directly instead. Maybe an additional `scrollContainer` option might be relevant for this. See an example for the the ResultList component (`infiniteScrollContainer` option) using the `ComponentOptions.findParentScrolling` method.

Reply:
Very good idea, I'll add that option.